aio/tools/transforms/angular-api-package/processors/mergeDecoratorDocs.spec.js

```diff
@@ -88,4 +87,23 @@ describe('mergeDecoratorDocs processor', () => {
     processor.$process([decoratorDoc, metadataDoc, otherDoc]);
     expect(decoratorDoc.docType).toEqual('decorator');
   });
+
+  it('should handle a type cast before the "make decorator" call', () => {
+    decoratorDoc.symbol.valueDeclaration.initializern = {
+      expression: decoratorDoc.symbol.valueDeclaration.initializer,
+      type: {},
+    };
+    processor.$process([decoratorDoc, metadataDoc, otherDoc]);
+    expect(decoratorDoc.docType).toEqual('decorator');
+  });
+
+  it('should handle the "make decorator" call being wrapped in a call to `attachInjectFlag()`',
+     () => {
+       decoratorDoc.symbol.valueDeclaration.initializern = {
+         expression: {test: 'attachInjectFlag'},
+         args: [decoratorDoc.symbol.valueDeclaration.initializer]
```

Comment:
Shouldn't these be:

```suggestion
         expression: {text: 'attachInjectFlag'},
         arguments: [decoratorDoc.symbol.valueDeclaration.initializer]
```

❓